mlos_core/mlos_core/spaces/__init__.py

```diff
@@ -85,3 +85,39 @@ def _one_parameter_convert(parameter: ConfigSpace.hyperparameters.Hyperparameter
         raise ValueError(f"Type of parameter {parameter} ({type(parameter)}) not supported.")
 
     return emukit.core.ParameterSpace([_one_parameter_convert(param) for param in config_space.get_hyperparameters()])
+
+
+def configspace_to_flaml_space(config_space: ConfigSpace.ConfigurationSpace) -> dict:
```

Comment:
Here's another one we might want to start splitting out to multiple files.
One reason to do this would be to adjust the way we handle imports so they can be top level instead of nested.
Separate PR  for that too ...

Reply:
Yep, happy to do this, once both PRs are merged! :)